src/main/java/edu/harvard/h2ms/domain/core/Event.java

```diff
@@ -1,94 +1,115 @@
-package edu.harvard.h2ms.domain.core;
-
-import javax.persistence.*;
-import java.sql.Time;
-
-/**
- * An Event is...
- */
-@Entity
-@Table(name = "EVENT")
-public class Event {
-
-    /* Properties */
-    private Long id;
-    private Time timestamp;
-    private String handWashType;
-    private String relativeMoment;
-    private String observee;
-    private String observer;
-    private String observationType;
-
-    @Id
-    @GeneratedValue(strategy= GenerationType.AUTO)
-    @Column(name = "ID")
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    @Column(name = "TIMESTAMP")
-    public Time getTimestamp() {
-        return timestamp;
-    }
-
-    public void setTimestamp(Time timestamp) {
-        this.timestamp = timestamp;
-    }
-
-    @Column(name = "HAND_WASH_TYPE")
-    public String getHandWashType() {
-        return handWashType;
-    }
-
-    public void setHandWashType(String handWashType) {
-        this.handWashType = handWashType;
-    }
-
-    @Column(name = "RELATIVE_MOMENT")
-    public String getRelativeMoment() {
-        return relativeMoment;
-    }
-
-    public void setRelativeMoment(String relativeMoment) {
-        this.relativeMoment = relativeMoment;
-    }
-
-    @Column(name = "OBSERVEE")
-    public String getObservee() {
-        return observee;
-    }
-
-    public void setObservee(String observee) {
-        this.observee = observee;
-    }
-
-    @Column(name = "OBSERVER")
-    public String getObserver() {
-        return observer;
-    }
-
-    public void setObserver(String observer) {
-        this.observer = observer;
-    }
-
-    @Column(name = "OBSERVATION_TYPE")
-    public String getObservationType() {
-        return observationType;
-    }
-
-    public void setObservationType(String observationType) {
-        this.observationType = observationType;
-    }
-
-    @Override
-    public String toString() {
-        return "Event - Id: " + id + ", Typestamp: " + timestamp
-                + ", Hand Wash Type: " + handWashType  + ", Relative Moment: " + relativeMoment
-                + ", Observee: " + observee  + ", Observer: " + observer  + ", Observation Type: " + observationType;
-    }
-
-}
+package edu.harvard.h2ms.domain.core;
+
+import java.sql.Time;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * An Event is what observer or sensor records about observee's actions.
+ */
+@Entity
+@Table(name = "EVENT")
+public class Event {
+
+    /* Properties */
+	
+	@Id
+	@GeneratedValue(strategy= GenerationType.AUTO)
+    @Column(name = "ID")
+    private Long id;
+	
+	@Column(name = "TIMESTAMP")
+    private Date timestamp;
+	
+	@Column(name = "METHOD_ID")
+    private Long method_id;

```

Comment:
I think you can use a real Method object here.  I tried it locally on my copy and it looks something like this:


```java
@ManyToOne(fetch = FetchType.LAZY)
@JoinColumn(name = "method_id")
private Method method;
```

then in Method:

```java
@OneToMany(targetEntity=Event.class
cascade = CascadeType.ALL,
fetch = FetchType.LAZY)
private Set<Event> events = new HashSet<>();
```

That's just based on what I saw in the Spring docs, I don't know if it's the right thing to do